### ISSUE TITLE
There is a small mistake in tabular_combiner.py

### ISSUE THREAD

**Issue by weibobo2015** [georgian-io/Multimodal-Toolkit#9]
As the title says, there is a mistake in tabular_combiner.py.
Line 409: 
g_num = (torch.cat([w_text, w_cat], dim=-1) * self.weight_a).sum(dim=1).unsqueeze(0).T

should change to:
g_num = (torch.cat([w_text, w_num], dim=-1) * self.weight_a).sum(dim=1).unsqueeze(0).T 

because it is in a numerical part which should use w_num instead of w_cat
